src/server/scripts/Northrend/Ulduar/Ulduar/boss_mimiron.cpp

```diff
@@ -1158,21 +1159,8 @@ class npc_ulduar_leviathan_mkii : public CreatureScript
                     events.ScheduleEvent(EVENT_PROXIMITY_MINES_1, 8000);
                     break;
                 case EVENT_PROXIMITY_MINES_1:
-                    {
-                        float angle = rand_norm() * 2 * M_PI;
-                        float x, y, z;
-                        me->GetPosition(x, y, z);
-                        for( uint8 i = 0; i < 17; ++i )
-                        {
-                            if( i == 7 )
-                                continue;
-
-                            float v_xmin = 0.1f * cos( angle + i * M_PI / 9 );
-                            float v_ymin = 0.1f * sin( angle + i * M_PI / 9 );
-                            if( Creature* pmNPC = me->SummonCreature(NPC_PROXIMITY_MINE, x + v_xmin, y + v_ymin, z, 0.0f, TEMPSUMMON_TIMED_DESPAWN, 40000) )
-                                pmNPC->KnockbackFrom(x, y, 6.0f, 25.0f);
-                        }
-                    }
+                    for (uint8 i = 0; i < 10; ++i)
+                    me->CastSpell(me, SPELL_SUMMON_PROXIMITY_MINE, true);
```

Comment:
```suggestion
                    for (uint8 i = 0; i < 10; ++i)
                    {
                        me->CastSpell(me, SPELL_SUMMON_PROXIMITY_MINE, true);
                    }
```

Reply:
Good catch, atm it isnt even in the for loop lol